packages/babel-plugin-transform-es3-reserved-words/src/index.js

```diff
@@ -0,0 +1,15 @@
+import { es3, es2017 } from "./reserved-words";
+
+export default function() {
+  const reservedKeywords = es3.filter(w => !(es2017.indexOf(w) !== -1));
```

Comment:
personally I would move this outside of the plugin function to the outer scope, if something is nested i'm always instantly searching for a reason why is it nested (basically what variables from the current closure are used) - and when it the things are top level is way more obvious that i can mentally just skip this step while reading

also what it does is a [`difference`](https://lodash.com/docs/4.17.4#difference) - im not saying we should add dependency on lodash (or any other util lib) here, but it would be beneficial to extract this function to give more meaning to whats going on here
```js
const difference = (arr, arr2) => arr.filter(el => !(arr2.indexOf(el) !== -1));
const reservedKeywords = difference(es3, es2017);
```

Reply:
I really like this, thank you! Changed it.